cpp/csp/adapters/websockets/ClientHeaderUpdateAdapter.h

```diff
@@ -0,0 +1,33 @@
+#ifndef _IN_CSP_ADAPTERS_WEBSOCKETS_CLIENT_HEADERUPDATEADAPTER_H
+#define _IN_CSP_ADAPTERS_WEBSOCKETS_CLIENT_HEADERUPDATEADAPTER_H
+
+#include <websocketpp/config/asio_client.hpp>
+#include <websocketpp/client.hpp>
+#include <csp/engine/Dictionary.h>
+#include <csp/engine/OutputAdapter.h>
+#include <csp/adapters/utils/MessageWriter.h>
+
+namespace csp::adapters::websockets
+{
+
+class ClientHeaderUpdateAdapter final: public OutputAdapter
+{
+public:
+    ClientHeaderUpdateAdapter(
+        Engine * engine,
+        Dictionary& properties
+    );
+
+    void executeImpl() override;
+
+    const char * name() const override { return "ClientHeaderUpdateAdapter"; }
```

Comment:
same comment on the rest of the name() methods, add websocket in the name somewhere